obspy/taup/velocity_model.py

```diff
@@ -484,6 +483,128 @@ def readTVelFile(cls, filename):
                              cls.default_cmb, cls.default_iocb, 0,
                              maxRadius, True, layers)
 
+    @classmethod
+    def read_nd_file(cls, filename):
+        """
+        This method reads in a velocity model from a "nd" ASCII text file, the
+        format used by Xgbm. The name of the model file for model "modelname"
+        should be "modelname.nd".
```

Comment:
The "Method to" is not necessary in the one line summary. Just say "Read in a velocity model ...".